diff -up xserver-master/hw/xfree86/common/xf86Init.c.12~ xserver-master/hw/xfree86/common/xf86Init.c
--- xserver-master/hw/xfree86/common/xf86Init.c.12~	2025-06-05 17:50:09.000000000 +0200
+++ xserver-master/hw/xfree86/common/xf86Init.c	2025-06-06 18:40:28.218159252 +0200
@@ -859,9 +859,14 @@ OsVendorFatalError(const char *f, va_lis
 {
     ErrorF("\nPlease consult the " XVENDORNAME " support \n\t at "
            __VENDORDWEBSUPPORT__ "\n for help. \n");
-    if (xf86LogFile && xf86LogFileWasOpened)
-        ErrorF("Please also check the log file at \"%s\" for additional "
-               "information.\n", xf86LogFile);
+    if (xf86LogFile && xf86LogFileWasOpened) {
+        if (strcmp(xf86LogFile, "/dev/null") == 0)
+            ErrorFSigSafe("Please also check `journalctl -b /usr/bin/Xorg` for "
+                         "additional information.\n");
+        else
+            ErrorFSigSafe("Please also check the log file at \"%s\" for additional "
+                         "information.\n", xf86LogFile);
+    }
     ErrorF("\n");
 }
 
