diff -urN xserver-xlibre-xserver-25.1.0/hw/xfree86/common/xf86Module.h xserver-xlibre-xserver-25.1.0-fix-abi-tests/hw/xfree86/common/xf86Module.h
--- xserver-xlibre-xserver-25.1.0/hw/xfree86/common/xf86Module.h	2025-12-21 14:53:30.000000000 +0000
+++ xserver-xlibre-xserver-25.1.0-fix-abi-tests/hw/xfree86/common/xf86Module.h	2026-01-11 19:53:09.680526837 +0000
@@ -74,14 +74,7 @@
  * mask is 0xFFFF0000.
  */
 #define ABI_ANSIC_VERSION	SET_ABI_VERSION(1, 4)
-
-/* XXX This is a compile-time option that changes abi XXX */
-/* TODO: Remove this toggle in 26.0 */
-#ifdef CONFIG_LEGACY_NVIDIA_PADDING
-#define ABI_VIDEODRV_VERSION	SET_ABI_VERSION(28, 1)
-#else
 #define ABI_VIDEODRV_VERSION    SET_ABI_VERSION(28, 0)
-#endif
 #define ABI_XINPUT_VERSION	SET_ABI_VERSION(26, 0)
 #define ABI_EXTENSION_VERSION	SET_ABI_VERSION(11, 0)
 
